packages/kolibri-common/strings/bulkUserManagementStrings.js

```diff
@@ -292,6 +308,10 @@ export const bulkUserManagementStrings = createTranslator('BulkUserManagementStr
     message: 'Assign to a class',
     context: 'Label for the classes input field in the create user modal',
   },
+  selectClassesLabel: {
+    message: 'Select classes',
```

Comment:
Just noting that we have other select classes translation https://github.com/ozer550/kolibri/blob/9ff8e08c51a5caff5389be762025b4d871aab542/packages/kolibri-common/strings/bulkUserManagementStrings.js#L65